### ISSUE TITLE
Normalize benchmarks

### ISSUE THREAD

**Issue by janden** [kymatio/kymatio#359]
Right now, the different `compute_speed.py` scripts have very different running times. We should normalize these so that they take about the same time. Of course, this has to be balanced with ensuring large enough batch sizes to ensure maximum efficiency.

**Comment by edouardoyallon** [comment]
Duplicate of #259

**Comment by janden** [comment]
> Duplicate of #259

I disagree. #259 is about including benchmarks into the CI process. This is about changing the benchmarks so that they're of comparable size.

**Comment by janden** [comment]
We're bringing this out of 0.2 and moving it to 0.3. The docs need to updated to reflect the new tests and the tests have to work for GPU, if available.
